vhdbuilder/packer/install-dependencies.sh

```diff
@@ -183,6 +188,16 @@ unpackAzureCNI() {
   echo "  - Ran tar -xzf on the CNI downloaded then rm -rf to clean up"
 }
 
+#this is the reference cni it is only ever downloaed in caching not at provisioning 
+#but conceptually it si very similiar to downloadAzureCNI in that it takes a url and puts in CNI_DOWNLOADS_DIR
+downloadCNI() {
```

Comment:
There is already a downloadCNI in cse_install.sh. Why duplicate?

Reply:
No longer used in cse_install.sh removed so I moved it closer to its only caller.